src/main/resources/data-submitter-schema_v1.json

```diff
@@ -0,0 +1,386 @@
+      }
+    },
+    "publicVisibility": {
+      "type": "boolean",
+      "label": "Public Visibility",
+      "description": "Public Visibility of this study",
+      "prompt": "Please select if you would like your dataset to be publicly visible for the requesters to see and select for an access request"
+    },
+    "nihAnvilUse": {
+      "type": "array",
+      "items": {
+          "type": "string",
+          "enum": ["I did", "I will", "No"]
+      }
+    },
+    "submittingToAnvil": {
+      "type": "boolean",
+      "label": "Are you planning to submit to AnVIL?",
+      "description": "Are you planning to submit to AnVIL?"
+    },
+    "dbGaPPhsID": {
+      "type": "string",
+      "label": "dbGaP phs ID",
+      "description": "dbGaP phs ID"
+    },
+    "dbGaPStudyRegistrationName": {
+      "type": "string",
+      "label": "dbGaP Study Registration Name",
+      "description": "dbGaP Study Registration Name"
+    },
+    "embargoReleaseDate": {
+      "type": "string",
+      "format": "date",
+      "label": "Embargo Release Date",
+      "description": "Embargo Release Date"
+    },
+    "sequencingCenter": {
+      "type": "string",
+      "label": "Sequencing Center",
+      "description": "Sequencing Center"
+    },
+    "piEmail": {
+      "type": "string",
+      "format": "email",
+      "label": "Principal Investigator Email",
+      "description": "Principal Investigator Email"
+    },
+    "piInstitution": {
+      "type": "number",
+      "label": "Principal Investigator Institution",
+      "description": "Principal Investigator Institution"
+    },
+    "nihGrantContractNumber": {
+      "type": "string",
+      "label": "NIH Grant or Contract Number",
+      "description": "NIH Grant or Contract Number"
+    },
+    "nihICsSupportingStudy": {
+      "type": "string",
+      "label": "NIH ICs Supporting the Study",
+      "description": "NIH ICs Supporting the Study",
+      "enum": ["NCI", "NEI", "NHLBI", "NHGRI", "NIA", "NIAAA", "NIAID", "NIAMS", "NIBIB", "NICHD", "NIDCD", "NIDCR", "NIDDK", "NIDA", "NIEHS", "NIGMS", "NIMH", "NIMHD", "NINDS", "NINR", "NLM", "CC", "CIT", "CSR", "FIC", "NCATS", "NCCIH"]
+    },
+    "nihProgramOfficerName": {
+      "type": "string",
+      "label": "NIH Program Officer Name",
+      "description": "NIH Program Officer Name"
+    },
+    "nihInstitutionCenterSubmission": {
+      "type": "string",
+      "label": "NIH Institution/Center for Submission",
+      "description": "NIH Institution/Center for Submission",
+      "enum": ["NCI", "NEI", "NHLBI", "NHGRI", "NIA", "NIAAA", "NIAID", "NIAMS", "NIBIB", "NICHD", "NIDCD", "NIDCR", "NIDDK", "NIDA", "NIEHS", "NIGMS", "NIMH", "NIMHD", "NINDS", "NINR", "NLM", "CC", "CIT", "CSR", "FIC", "NCATS", "NCCIH"]
+    },
+    "nihGenomicProgramAdministratorName": {
+      "type": "string",
+      "label": "NIH Genomic Program Administrator Name",
+      "description": "NIH Genomic Program Administrator Name"
+    },
+    "multiCenterStudy": {
```

Comment:
Not sure where all of the fields come from, but if `multiCenterStudy` was true, does this imply there are multiple NIH centers? If so, where would those be denoted?

Reply:
Those are defined in the next property, `collaboratingSites`. It's supposed to be an array of (free text) sites, but I'm not sure how the UI will be able to supply that at the moment.